src/Core/Resolvers/SqlMutationEngine.cs

```diff
@@ -156,6 +161,33 @@ public class SqlMutationEngine : IMutationEngine
                             result = GetDbOperationResultJsonDocument("success");
                         }
                     }
+                    // This code block contains logic for handling multiple create mutation operations.
+                    else if (mutationOperation is EntityActionOperation.Create && _runtimeConfigProvider.GetConfig().IsMultipleCreateOperationEnabled())
+                    {
+                        bool isPointMutation = IsPointMutation(context);
+
+                        List<IDictionary<string, object?>> primaryKeysOfCreatedItems = PerformMultipleCreateOperation(
+                                    entityName,
+                                    context,
+                                    parameters,
+                                    sqlMetadataProvider,
+                                    !isPointMutation);
+
+                        if (isPointMutation)
```

Comment:
Where do we have the non-multiple create mutation logic? what stops a pointMutation (non mulitple create) from hitting this logic?

Reply:
For processing the selection set, the logic for a point multiple create operation and a create mutation operation (not multiple create) is the same. Many type multiple create mutation operation follows a different logic for resolving the selection set. 

Hence, it is sufficient to distinguish based on the whether the create mutation operation is point vs many type

Reply:
*When Multiple Create is disabled:*

When the feature flag for multiple create mutation is disabled, create mutation operation will continue to follow the current logic that we have today. A check for this is performed few lines above the if condition: `if(isPointMutation)`

*When Multiple Create is enabled:*

When multiple create feature is enabled, both "legacy create mutation"(create mutation without inputs for related entity) and "multiple create mutation" will follow the same code path. 
So, the `if(isPointMutation)` condition will be hit by both legacy create mutation as well as multiple create mutation.